native/common/include/jpype.h

```diff
@@ -208,11 +209,4 @@ typedef _object PyObject;
 #include "jp_primitivetype.h"
 #include "jp_typemanager.h"
 
-// Accessors
-#include "jp_array.h"
-#include "jp_buffer.h"
-
-// Services
-#include "jp_proxy.h"
-
-#endif // _JPYPE_H_
```

Comment:
There was  a lot of global includes.  This PR finished a lot of that clean up work by pushing the last few specializazed wrappers out of the global space.